### PR TITLE
[pi-ui][settings] Use pi-ui's Checkbox component instead of custom one

### LINKED ISSUE #2661: [pi-ui][settings] Use pi-ui's Checkbox component instead of custom one.
image

### PR DESCRIPTION
**Pull request by amassarwi**
This diff replaces custom checkbox used in privacy settings with pi-ui's component.
Closes #2661 
Needs decred/pi-ui#294
~~Based on #2662~~

---

image

### CODE REVIEW

**Review by vctt94 — APPROVED** [review]
Thanks for this PR. It was a pain to work with checkboxes on decrediton!

File: package.json [hunk 1]
```diff
@@ -283,7 +283,7 @@
     "node-abi": "^2.15.0",
     "node-addon-loader": "decred/node-addon-loader#master",
     "nouislider": "^12.0.0",
-    "pi-ui": "https://github.com/decred/pi-ui",
+    "pi-ui": "https://github.com/amassarwi/pi-ui#checkboxAdj",
```
Comment (amassarwi) [inline package.json:286]: Will revert when pi-ui's PR is in

File: yarn.lock [hunk 2]
```diff
@@ -9319,9 +9319,9 @@ performance-now@^2.1.0:
   resolved "https://registry.yarnpkg.com/performance-now/-/performance-now-2.1.0.tgz#6309f4e0e5fa913ec1c69307ae364b4b377c9e7b"
   integrity sha1-Ywn04OX6kT7BxpMHrjZLSzd8nns=
 
-"pi-ui@https://github.com/decred/pi-ui":
+"pi-ui@https://github.com/amassarwi/pi-ui#checkboxAdj":
   version "1.0.0"
-  resolved "https://github.com/decred/pi-ui#c9741462323ab7d7c30059827f8b618f7bfc5b8d"
+  resolved "https://github.com/amassarwi/pi-ui#b124cc03cc5087a533a86850dd9f5a96b092250b"
```
Comment (amassarwi) [inline yarn.lock:9324]: Will revert when pi-ui's PR is in

Comment (alexlyp) [reply yarn.lock:9324]: Hrm would prefer to wait on this one until pi-ui is up to date.

Comment (amassarwi) [reply yarn.lock:9324]: of course, commented to remind myself what to revert before merging.

Comment (alexlyp) [reply yarn.lock:9324]: gotcha, looks like it's failing nonetheless

Comment (amassarwi) [reply yarn.lock:9324]: yep, rebase smoke 🚒 🧯 🧑‍🚒

**Comment by amassarwi** [comment]
@alexlyp pi-ui bumped it should be ready to go now.

**Review by alexlyp — APPROVED** [review]
ACK

**Pull request merged**